docs/UserGuide.md

```diff
@@ -297,13 +297,15 @@ Examples:
 
 ### Redoing an Undo : `redo`
 
-If the previous command was an `undo`, then will restore EmployeeManager to the state before `undo`.
+Restores EmployeeManager to the state before the last `undo` as long as no command that made a modification to the data in EmployeeManager was executed since the last `undo`.
```

Comment:
Would this be clearer?
Restores EmployeeManager to its previous state before the most recent undo, provided that no commands altering EmployeeManager's data have been executed since that undo.